pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -355,18 +373,16 @@ const MemoryRow = ({ memorySize, memorySizeUnit, nodeMaxMemory, onValueChanged,
                     initialUnit={memorySizeUnit}
                     onValueChange={e => onValueChanged('memorySize', e.target.value)}
                     onUnitChange={value => onValueChanged('memorySizeUnit', value)} />
-                {validationStateMemory === "error" &&
-                <HelpBlock>
-                    <p className="text-danger">{validationFailed.memory}</p>
-                </HelpBlock> }
-                {nodeMaxMemory &&
                 <HelpBlock id="memory-size-helpblock">
-                    {cockpit.format(
-                        _("Up to $0 $1 available on the host"),
-                        Math.floor(convertToUnit(nodeMaxMemory, units.KiB, memorySizeUnit)),
-                        memorySizeUnit,
-                    )}
-                </HelpBlock>}
+                    <p>{validationStateMemory === "error" && validationFailed.memory}</p>
```

Comment:
A nit-pick, but perhaps let's change order of these helpblocks. We show memory related to minimal required memory being higher, then we show available memory on host, and then recomemmed. It would make more sense on order being: minimal memory, recommended memory, available on host.

Reply:
Done